src/api/init.ts

```diff
@@ -225,7 +226,7 @@ export async function startApiServer(opts: {
       router.use('/fee_rate', createFeeRateRouter(datastore));
       router.use('/tokens', createTokenRouter(datastore));
       router.use('/pox2_events', createPox2EventsRouter(datastore));
-      if (chainId !== ChainID.Mainnet && writeDatastore) {
+      if (chainId !== ChainID.Mainnet && writeDatastore && BTC_RPC_PORT && BTC_RPC_HOST) {
```

Comment:
Yeah. You are right this will disable all endpoints for /faucets. I have fixed it and the current change will only disable /faucets/btc and /faucet/btc/:address endpoints. Kindly look into it?